tools/PostProcessing/read_module.f90

```diff
@@ -458,10 +458,10 @@ subroutine read_eigenvalues
     efermi = zero
     if(nspin==1) then
        read(17,fmt='(a6,f18.10)') str,efermi(1)
-       write(*,fmt='(4x,"Fermi level: ",f12.5," Ha")') efermi(1)
+       write(*,fmt='(4x,"Fermi level: ",f12.5," Ha   (=",f7.3," eV)")') efermi(1), efermi(1)*HaToeV
     else
        read(17,fmt='(a6,2f18.10)') str,efermi(1), efermi(2)
-       write(*,fmt='(4x,"Fermi levels: ",2f12.5," Ha")') efermi
+       write(*,fmt='(4x,"Fermi levels: ",2f12.5," Ha   (=",2f7.3" eV)")') efermi, efermi*HaToeV
```

Comment:
Same formatting question here: maybe `f10.3` might be safer.

Reply:
Thank you for your comment.
I modified them to f10.3.